ampd/Dockerfile

```diff
@@ -22,6 +22,7 @@ COPY ./ampd/build.rs ./ampd/build.rs
 RUN cargo install --locked --path ./ampd
 
 FROM debian:bookworm-slim AS runner
+RUN apt update && apt install libssl3
```

Comment:
why is this needed now? There's already an apt update and install at the top

Reply:
Looks like the previous apt install is for a different image

Reply:
yes, currently we are unable to execute ampd in the runner image because libssl3 is missing.

Reply:
maybe because we don't pin the bookworm-slim version